frontend/src/components/landing/Reviews.module.scss

```diff
@@ -0,0 +1,294 @@
+@import "../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  display: grid;
+  justify-items: center;
+
```

Comment:
I might not be looking right, but these don't appear to do anything?

```suggestion
```

Reply:
removed